File: ViewModels/Project.cs
using System;

namespace HonestProject.ViewModels
{
    public class Project{
        public Guid Id {get; set;}
        public string Name {get; set;}
        public string Description {get; set;}
        public DateTime StartDate {get; set;}
        public int PercentageEstimate {get; set;}
        public string Color {get; set;}
        public ProjectWorkType[] WorkTypes {get; set;}
    }
}

File: src/app/landingPage/models/Project.ts
import { ProjectWorkType } from "./ProjectWorkType";

export class Project{
    constructor(
        public id: string,
        public name: string,
        public description: string,
        public startDate: Date,
        public percentageEstimate: number,
        public color: string,
        public teamId: string,
        public workTypes: ProjectWorkType[]
    ) {}
}